ttnn/cpp/ttnn/operations/eltwise/binary/common/binary_op_utils.cpp

```diff
@@ -191,8 +191,13 @@ std::map<std::string, std::string> get_defines_fp32(
             }
             break;
         case BinaryOpType::SUB:
-            new_defines.insert({"BINOP_INIT", fmt::format("sub_binary_tile_init();")});
-            op_name = "sub_binary_tile";
+            if (input_a_dtype == DataType::INT32 && input_b_dtype == DataType::INT32) {
+                new_defines.insert({"SUB_INT32_INIT", fmt::format("sub_int32_tile_init();")});
+                op_name = "sub_int32_tile";
+            } else {
+                new_defines.insert({"BINOP_INIT", fmt::format("sub_binary_tile_init();")});
+                op_name = "sub_binary_tile";
+            }
```

Comment:
I understand the pattern was already there, but at least within these changes can you remove the unnecessary calls to `fmt::format`?